src/components/exchange/ConfirmExchangeButton.js

```diff
@@ -107,13 +107,13 @@ export default function ConfirmExchangeButton({
     label = 'Insufficient Funds';
   } else if (!isSufficientLiquidity) {
     label = 'Insufficient Liquidity';
-  } else if (!isSufficientGas) {
+  } else if (gasPrices && !isSufficientGas) {
     label = 'Insufficient ETH';
```

Comment:
Looks redundant

Reply:
isSufficientGas can be falsey initially when gasPrices do not exist (as we are removing the fallback)!